test_asan_bg.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+pids=()
+exit_codes=()
+parallel_tags="[diskann] [hnsw]"
+othertag=""
+
+./build/tests/unittests -d yes ${UT_FILTER} --allow-running-no-tests --shard-count 1 --shard-index 0 &
+pids+=($!)
+
+for tag in ${parallel_tags}
+do
+  othertag="~"${tag}${othertag}
+  ./build/tests/functests -d yes ${UT_FILTER} --allow-running-no-tests ${tag} &
+  pids+=($!)
+done
+
+./build/tests/functests -d yes ${UT_FILTER} --allow-running-no-tests ${othertag} &
+pids+=($!)
+
+for pid in "${pids[@]}"
+do
+  wait $pid
+  exit_codes+=($?)
+done
```

Comment:
what is the behavior here when the second process exit before the first process

Reply:
no problem, I have checked